### PR TITLE
Feature/rb5009 u pr+s+in

### PR DESCRIPTION
**Pull request by prudy**
Hi, some changes, if you willing to get them in the way they are.
Before this decision please confirm if there is no regression to initially supported boards.
If ok, the openwrt Makefile must be updated to point to your repo.

### CODE REVIEW

File: mk_boards.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		.ports_num = 4,
+		.port_state_map = {0xd, 0xc, 0xb, 0xa}
+	}, { /* RouterBOARD 960PGS */
+		.name = "mikrotik,routerboard-960pgs",
+		.proto_ver = 3,
+		.spidev = "/dev/spidev0.2",
+		.ports_num = 4,
+		.port_state_map = {0xd, 0xc, 0xb, 0xa}
+	}, { /* RouterBOARD RB5009UPr+S+IN */
+		.name = "mikrotik,rb5009upr",
```
Comment (pktpls) [inline mk_boards.h:35]: Some RB5009 branches call it just `mikrotik,rb5009` - I'm using Arie's branch rebased on current main: https://github.com/Arie/openwrt-rb5009/tree/main

File: mtpoe_ctrl.h [hunk 2]
```diff
@@ -1,12 +1,14 @@
 #ifndef _MTPOE_CTRL
 #define _MTPOE_CTRL
 
+#include "mk_boards.h"
+
 //файл для общения с spidev модулем
 #define DEFAULT_DEV_FILE "/dev/spidev0.2"
 //ключ субконфига uci->network в котором находятся настройки PoE
-#define MTIK_POE_UCI_CONFIG_KEY "mtik_poe"
+#define MTIK_POE_UCI_CONFIG_KEY "@poe[0]"
```
Comment (pktpls) [inline mtpoe_ctrl.h:9]: I needed to change this to just `poe`, otherwise `load_poe_from_uci` wouldn't find the config entries

**Comment by prudy** [comment]
Hi,
This @poe[0] should be fine as if there is any other entry (if ever) it would not conflict. I bet that adding the 'poe' entry manually it would create ***@***.***[0]'. Could the mismatch be due to the other openwrt wersions?

Another thing I noticed yesterday is that new routeros rc updates the atmega chaging the mapping. This is quite odd but I must check the reported fw version.

**Comment by prudy** [comment]
I'm closing this in favor of the new one created.

**Pull request closed without merging**